### DISCUSSION TITLE
Question about updating superset-frontend

### DISCUSSION THREAD

**Discussion by Reslan-Tinawi** [apache-superset/superset-ui#1017] [Questions] [answered]
Hello superset community :wave: 

A couple of days ago, I created this [PR](https://github.com/apache-superset/superset-ui/pull/1000) to add new map to the country map plugin.

My question is, what should I do to update the packages I have for superset? I'm using the [docker installation](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) of superset.
I tried to update npm packages with `npm update` but nothing changed.

Should I wait till the next release of the packages? and is there a way that I can *force* npm to use the package with the new modification of my PR, so I don't have to wait to the release?

**Comment by Reslan-Tinawi** [comment]
@ktmud it'd be great if you can help :grinning: 
Thanks :pray:

**Comment by ktmud** [comment, marked as answer]
Someone needs to manually release the packages and create a PR in `superset` to bump the package versions ([example](https://github.com/apache/superset/pull/13593)). Generally if you don't want to wait for the release, you can fork the `superset-ui` repo and publish the packages under another name---then update the `package.json` in `superset/superset-frontend` to point to your own release.

**Reply by Reslan-Tinawi** [reply]
Thanks a lot for your detailed answer :pray: 

Someone had already released a new version of `superset-ui`: [publish v0.17.22 ](https://github.com/apache-superset/superset-ui/commit/6cb92e82ffa360841c9db6e42bfbbaa3cd74d26a), I just need to update my local npm packages.
